plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -2859,6 +2859,9 @@ public Map<SchemaTableName, ConnectorViewDefinition> getViews(ConnectorSession s
                 else if (e.getErrorCode().equals(HIVE_INVALID_VIEW_DATA.toErrorCode())) {
                     // Ignore views that are not valid
                 }
+                else if (e.getErrorCode().equals(HIVE_UNSUPPORTED_FORMAT.toErrorCode())) {
+                    // Ignore views that are not supported
```

Comment:
This is good to ignore this.
But we should also remove `throw e` below and replace with maybe a log.warn.
Just like we do in `io.trino.metadata.MetadataManager#handleListingError`